packages/webapp/src/containers/Finances/NewExpense/ExpenseCategories/index.jsx

```diff
@@ -27,7 +28,15 @@ export const icons = {
   SEEDS: <SeedIcon />,
   OTHER: <OtherIcon />,
   LAND: <LandIcon />,
-  MISCELLANEOUS: <OtherIcon />,
+  MISCELLANEOUS: (
+    <MiscellaneousIcon
+      style={{
+        border: 'solid 10px transparent',
+        filter:
+          'invert(30%) sepia(94%) saturate(787%) hue-rotate(136deg) brightness(103%) contrast(98%)',
```

Comment:
I believe this icon is temporary, but this is to change the colour of the svg.
https://stackoverflow.com/questions/22252472/how-can-i-change-the-color-of-an-svg-element